src/index.html

```diff
@@ -1,13 +1,508 @@
+                </li>
+                <li class="schedule__item">
+                  <span class="schedule__item--strong">12:00 — 20:00</span> п’ятниця
+                </li>
+                <li class="schedule__item">
+                  <span class="schedule__item--strong">11:00 — 19:00</span> субота
+                </li>
+                <li class="schedule__item">
+                  <span class="schedule__item--strong">11:00 — 19:00</span> неділя
+                </li>
+              </ul>
+              <p class="schedule__weekend schedule__weekend--footer">
+                <span class="schedule__weekend--strong-footer">Вихідні</span> понеділок, вівторок<br> Каса припиняє роботу за 1
+                годину до закриття музею
+              </p>
+            </div>
+            <address class="contacts footer__contact">
+              <p class="contacts__title">Контакти:</p>
+              <ul class="contacts__list">
+                <li class="contacts__item">
+                  <a
+                    href="https://www.google.com.ua/maps/place/%D1%83%D0%BB.+%D0%9C%D0%B8%D1%85%D0%B0%D0%B8%D0%BB%D0%B0+%D0%93%D1%80%D1%83%D1%88%D0%B5%D0%B2%D1%81%D0%BA%D0%BE%D0%B3%D0%BE,+6,+%D0%9A%D0%B8%D0%B5%D0%B2,+02000/@50.4494431,30.5281844,17z/data=!3m1!4b1!4m6!3m5!1s0x40d4ce52445a8315:0x5e30ef474eddac89!8m2!3d50.4494397!4d30.5307593!16s%2Fg%2F11l76wtstx?entry=ttu"
+                    class="contacts__data"
+                    target="_blank"
+                  >
+                    вул. М. Грушевського, 6; Київ, 01001
+                  </a>
+                </li>
+                <li class="contacts__item">
+                  <a href="tel:+38 (044) 278-13-57"
+                    class="contacts__data"
+                  >
+                    тел. +38 (044) 278-13-57
+                  </a>
+                </li>
+                <li class="contacts__item">
+                  <a
+                    href="tel:+38 (044) 278-74-54"
+                    class="contacts__data"
+                  >
+                    факс +38 (044) 278-74-54
+                  </a>
+                </li>
+                <li class="contacts__item">
+                  <a
+                    href="mailto:info@namu.kiev.ua"
+                    class="contacts__data"
+                  >
+                    e-mail: info@namu.kiev.ua
+                  </a>
+                </li>
+              </ul>
+            </address>
+          </div>
+          <div class="footer__menu">
+            <button
+              class="footer__scrollTop"
+              type="button"
+              data-footer-button="scroll-top"
+            >
+              <svg
+                class="icon icon__top footer__scrollTop-icon"
+                width="16"
+                height="16"
+                aria-label="Прокрутка до верху сторінки"
+              >
+                <use href="./images/icons/sprite.svg#arrow-up"></use>
+              </svg>
+            </button>
```

Comment:
Add some text to improve UX/UI. To clarify what this arrow does to user
![image](https://github.com/mate-academy/Museum/assets/97075591/6934213a-73f1-4d76-9f11-069899d448f5)

Reply:
I was making this element with design, so I don't think that some text need